jaeger-extension-ballerina/Ballerina.toml

```diff
@@ -19,6 +19,9 @@ org = "ballerinax"
 name = "jaeger"
 version = "@toml.version@"
```

Comment:
We have to add the `distribution = 2201.7.0` here
Better to have a variable

Reply:
added.